judge/judgedaemon.main.php

```diff
@@ -647,7 +619,7 @@ private function judging_directory(string $workdirpath, array $judgeTask): strin
             . $judgeTask['jobid'];
     }
 
-    private function read_credentials(): void
+    private function readCredentials(): void
```

Comment:
Nitpick: I'd be consistent and leave this as snake case as is.

Reply:
I did the opposite and remove all snake case from this file (as camel case is the best practice, and phpcs complains)